server/chat-commands.js

```diff
@@ -2723,8 +2723,8 @@ const commands = {
 		if (!reason) {
 			return this.errorReply(`Battle bans require a reason.`);
 		}
-		if (!room.battle && (!reason.includes('.pokemonshowdown.com/') && cmd !== 'forcebattleban')) {
-			 return this.errorReply(`Battle bans require a battle replay if used outside of a battle; if the battle has expired, use /forcebattleban.`);
+		if (!room.battle && (!reason.includes('.pokemonshowdown.com/') && cmd !== 'forcebattleban')) { // lgtm[js/incomplete-url-substring-sanitization]
```

Comment:
this should not throw a warning because the check is if it _doesn't_ include that part